index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    addToUI(book);
+  }
+
+  removeBook(id) {
+    const book = document.getElementById(id);
+    book.remove();
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+  }
```

Comment:
- [ ] Good job implementing the `addBook` function 👍 however when removing 1 book from a list of books it removes the entire list, to me, it looks like this is due to the fact that you are removing it based on an `id` however the books never get an `id` assigned.

Reply:
Thanks so much for the review. Let me make the changes :recycle: